policy_definitions/update_management/policy_definition_qby_deploy_maintenance_resources.json

```diff
@@ -0,0 +1,298 @@
+                "in": "[parameters('osTypes')]"
+              }
+            ]
+          }
+        ]
+      },
+      "then": {
+        "effect": "DeployIfNotExists",
+        "details": {
+          "type": "Microsoft.Maintenance/configurationAssignments",
+          "name": "[field(concat('tags[Severity Group Monthly]'))]",
+          "ExistenceScope": "Subscription",
+          "existenceCondition": {
+            "allOf": [
+              {
+                "field": "name",
+                "equals": "[field(concat('tags[Severity Group Monthly]'))]"
+              }
+            ]
+          },
+          "roleDefinitionIds": [
+            "/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c"
+          ],
+          "DeploymentScope": "Subscription",
+          "deployment": {
+            "location": "westeurope",
+            "properties": {
+              "mode": "Incremental",
+              "parameters": {
+                "severityGroupName": {
+                  "value": "[field(concat('tags[Severity Group Monthly]'))]"
+                },
+                "managementSubscriptionId": {
+                  "value": "[parameters('managementSubscriptionId')]"
+                },
+                "managementResourceGroup": {
+                  "value": "[parameters('managementResourceGroup')]"
+                },
+                "location": {
+                  "value": "[parameters('location')]"
+                },
+                "duration": {
+                  "value": "[parameters('duration')]"
+                },
+                "timeZone": {
+                  "value": "[parameters('timeZone')]"
+                },
+                "osTypes": {
+                  "value": "[parameters('osTypes')]"
+                },
+                "resourceTypes": {
+                  "value": "[parameters('resourceTypes')]"
+                }
+              },
+              "template": {
+                "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
+                "contentVersion": "1.0.0.0",
+                "parameters": {
+                  "severityGroupName": {
+                    "type": "string",
+                    "metadata": {
+                      "description": "Name of the severity group/ maintenance configuration."
+                    }
+                  },
+                  "managementSubscriptionId": {
+                    "type": "string",
+                    "metadata": {
+                      "description": "Subscription ID of the management subscription."
+                    }
+                  },
+                  "managementResourceGroup": {
+                    "type": "string",
+                    "metadata": {
+                      "description": "Name of the management resource group. The maintenance configuration will be deloyed there."
```

Comment:
Should we consider deploying all maintenance configurations in a separate RG? I think we did the same for DNS zones in connectivity to avoid overcrowding the one central RG.